### DISCUSSION TITLE
NiceHash in cluster.conf

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1172] [Q&A]
As I am looking for undocumented cluster options I stumbled upon this in ClusterConfig.cs: 
public class NicehashClusterConfig
{
    /// <summary>
    /// If set to true, the Nicehash service will be started
    /// </summary>
    public bool EnableAutoDiff { get; set; }
}

What does this function do exactly? And is it still in use?

kind regards

ei8ht

**Comment by oliverw** [comment]
It will query Nicehash's API for the required minimum difficulty for the coin in question.
